pyaedt/modules/SolveSweeps.py

```diff
@@ -0,0 +1,1067 @@
+from pyaedt.modules.SetupTemplates import NSSM
+from pyaedt.modules.SetupTemplates import PMDC
+from pyaedt.modules.SetupTemplates import SPIM
+from pyaedt.modules.SetupTemplates import SRM
+from pyaedt.modules.SetupTemplates import TPIM
+from pyaedt.modules.SetupTemplates import TPSM
+from pyaedt.modules.SetupTemplates import TR
+from pyaedt.modules.SetupTemplates import UNIM
+from pyaedt.modules.SetupTemplates import Close
+from pyaedt.modules.SetupTemplates import EddyCurrent
+from pyaedt.modules.SetupTemplates import ElectricTransient
+from pyaedt.modules.SetupTemplates import Electrostatic
+from pyaedt.modules.SetupTemplates import HFSS3DLayout
+from pyaedt.modules.SetupTemplates import HFSSDrivenAuto
+from pyaedt.modules.SetupTemplates import HFSSDrivenDefault
+from pyaedt.modules.SetupTemplates import HFSSEigen
+from pyaedt.modules.SetupTemplates import HFSSTransient
+from pyaedt.modules.SetupTemplates import LNA3DLayout
+from pyaedt.modules.SetupTemplates import Magnetostatic
+from pyaedt.modules.SetupTemplates import Matrix
+from pyaedt.modules.SetupTemplates import MaxwellTransient
+from pyaedt.modules.SetupTemplates import MechModal
+from pyaedt.modules.SetupTemplates import MechStructural
+from pyaedt.modules.SetupTemplates import MechTerm
+from pyaedt.modules.SetupTemplates import NexximAMI
+from pyaedt.modules.SetupTemplates import NexximDC
+from pyaedt.modules.SetupTemplates import NexximHarmonicBalance1T
+from pyaedt.modules.SetupTemplates import NexximHarmonicBalanceNT
+from pyaedt.modules.SetupTemplates import NexximLNA
+from pyaedt.modules.SetupTemplates import NexximOscillator1T
+from pyaedt.modules.SetupTemplates import NexximOscillatorNT
+from pyaedt.modules.SetupTemplates import NexximOscillatorRSF
+from pyaedt.modules.SetupTemplates import NexximQuickEye
+from pyaedt.modules.SetupTemplates import NexximSystem
+from pyaedt.modules.SetupTemplates import NexximTransient
+from pyaedt.modules.SetupTemplates import NexximTVNoise
+from pyaedt.modules.SetupTemplates import NexximVerifEye
+from pyaedt.modules.SetupTemplates import Open
+from pyaedt.modules.SetupTemplates import SiwaveAC3DLayout
+from pyaedt.modules.SetupTemplates import SiwaveDC3DLayout
+from pyaedt.modules.SetupTemplates import SteadyFlowOnly
+from pyaedt.modules.SetupTemplates import SteadyTemperatureAndFlow
+from pyaedt.modules.SetupTemplates import SteadyTemperatureOnly
+from pyaedt.modules.SetupTemplates import Sweep3DLayout
+from pyaedt.modules.SetupTemplates import SweepHfss3D
+from pyaedt.modules.SetupTemplates import SweepSiwave
+from pyaedt.modules.SetupTemplates import TransientFlowOnly
+from pyaedt.modules.SetupTemplates import TransientTemperatureAndFlow
+from pyaedt.modules.SetupTemplates import TransientTemperatureOnly
+
+open3 = open
+if sys.version_info < (3, 0):
+    import io
+
+    open3 = io.open
+
+
+@pyaedt_function_handler()
+def identify_setup(props):
+    """Identify if a setup props is time or frequency domain.
```

Comment:
```suggestion
    """Identify if a setup's properties is based on a time or frequency domain.
```
I'm guessing on the meaning intended here.  You might need to clarify. You also need a description for the ``props`` parameter.